pymatviz/ptable.py

```diff
@@ -699,292 +730,438 @@ def histogram(
         ax.spines[["right", "top", "left"]].set_visible(False)
 
 
-def ptable_heatmap(
-    values: ElemValues,
-    *,
-    log: bool | Normalize = False,
-    ax: plt.Axes | None = None,
-    count_mode: ElemCountMode = ElemCountMode.composition,
-    cbar_title: str = "Element Count",
-    cbar_range: tuple[float | None, float | None] | None = None,
-    cbar_coords: tuple[float, float, float, float] = (0.18, 0.8, 0.42, 0.05),
-    cbar_kwargs: dict[str, Any] | None = None,
-    colorscale: str = "viridis",
-    show_scale: bool = True,
-    show_values: bool = True,
-    infty_color: str = "lightskyblue",
-    na_color: str = "white",
-    heat_mode: Literal["value", "fraction", "percent"] | None = "value",
-    fmt: str | Callable[..., str] | None = None,
-    cbar_fmt: str | Callable[..., str] | None = None,
-    text_color: str | tuple[str, str] = "auto",
-    exclude_elements: Sequence[str] = (),
-    zero_color: str = "#eff",  # light gray
```

Comment:
I think we might just migrate this `zero_color` and `zero_symbol` to `ptable_heatmap_ratio` as I think zero/inf should be a common case for ratio plotters (perhaps not the ptable heatmap plotter). Perhaps also add a `zero_tolerance` arg?

 If they user really want to overwrite the color for zero, they could use `overwrite_colors` to overwrite that.